src/libraries/System.Text.RegularExpressions/tests/FunctionalTests/RegexExperiment.cs

```diff
@@ -132,22 +97,37 @@ public void ViewSampleRegexInDGML()
                 return;
             }
 
+            if (!Directory.Exists(DgmlOutputDirectoryPath))
+            {
+                Directory.CreateDirectory(DgmlOutputDirectoryPath);
+            }
+
             try
             {
-                //string rawregex = @"\bis\w*\b";
-                string rawregex = And(".*[0-9].*[0-9].*", ".*[A-Z].*[A-Z].*", Not(".*(01|12).*"));
-                //string rawregex = "a.{4}$";
-                Regex re = new Regex($@"{rawregex}", RegexHelpers.RegexOptionNonBacktracking | RegexOptions.Singleline);
-                ViewDGML(re);
-                ViewDGML(re, inReverse: true);
-                ViewDGML(re, addDotStar: true);
-                ViewDGML(re, asNFA: true, bound: 12);
-                ViewDGML(re, inReverse: true, asNFA: true, bound: 12);
-                ViewDGML(re, addDotStar: true, asNFA: true, bound: 12);
+                /*lang=regex*/
```

Comment:
NIT: Perhaps it would be good to add some comments to this test which more or less explains what is the test for, and how it should be used? E.g, saying that you need to flip Enabled bool, and where are the dgmls generated at and how to open them in VS?

Reply:
ok